pkg/agent/qrm-plugins/cpu/nativepolicy/util/util.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/kubewharf/katalyst-core/pkg/util/machine"
 )
 
+// GenerateMachineStateFromPodEntries for native policy
 func GenerateMachineStateFromPodEntries(topology *machine.CPUTopology, podEntries state.PodEntries) (state.NUMANodeMap, error) {
```

Comment:
Begin with a lowercase letter like `generateMachineStateFromPodEntries`?

Reply:
@luomingmeng NewNativePolicy function in package nativepolicy also need to call GenerateMachineStateFromPodEntries function